src/sage/graphs/generic_graph.py

```diff
@@ -15746,6 +15746,93 @@ def distance_all_pairs(self, by_weight=False, algorithm=None,
                                             weight_function=weight_function,
                                             check_weight=check_weight)[0]
 
+    def power(self, k):
+        r"""
+        Compute the kth power graph of an unweighted graph based on shortest
+        distances between nodes using BFS.
+
+        INPUT:
+
+        - ``k`` -- integer; the maximum path length for considering edges in the power graph.
```

Comment:
When the description of a parameter is on multiple lines, the second line has 2 spaces more.
```py
        - ``k`` -- integer; the maximum path length for considering edges in the
          power graph.
```
See other methods for examples.